src/components/AddNote.jsx

```diff
@@ -20,7 +30,10 @@ export default function AddNote() {
     } catch (error) {
       console.log(error)
     }
+  } else {
+    Toast({type: "error", message:"Please login"})
```

Comment:
Suggestion: name it better something like showToast / addToast
also you should show a toast after updating note in line 26